agent/agent-plugins/redis-jedis-3.x/src/main/java/org/bithon/agent/plugin/redis/jedis3/interceptor/OnCommand.java

```diff
@@ -82,9 +78,10 @@ public void after(AopContext aopContext) {
             span.tag(aopContext.getException()).finish();
         }
 
-        JedisContext ctx = InterceptorContext.remove("redis-command");
+        JedisContext ctx = InterceptorContext.getAs("redis-command");
```

Comment:
why is this change needed?

Reply:
If remove is used, the InterceptorContext will no longer have the context information related to redis-command. Therefore, it needs to be modified to obtain the redis-command context based on the key, and then remove it after the statistics are completed.

Reply:
The `remove` removes the object from current thread local and returns the context object, I think it's the same with  the code you are changing. do I miss sth?